javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+
+function filterOutNulls(values) {
+  // debugger;
+  const newArray = [];
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] == null);
+    else {
+      newArray.push(values[i]);
+    }
```

Comment:
2 things here.

1. Your syntax is off here a bit. You're missing brackets after your `if` statement. It should be like
```
if (values[i] === null) {
} else {
```
2. Since you're not actually doing anything inside of your `if` statement, what's the point of having it? Right now, you are only running code inside the `else` statement. Why not change your `if` conditional so that code would run there? You can check if the current value is not equal to null instead.
```
if (values[i] !== null) {
   newArray.push(values[i]);
}
```